Updated for GA SDK 3.0 and 64 bit compatibility

mparrett opened the pull request:
This was a hasty update and I would like to have made smaller commits. This shouldn't be considered complete but may save someone some time. Specifically, transaction/eCommerce tracking was ignored. Let me know if you have any questions.

Commit:
Updated for GA SDK 3.0 and 64 bit compatibility

* 64bit/Universal compatible build
* Update Google Analytics SDK 2.0 -> 3.0
* Built with Flex SDK 4.9.1, Android SDK 24.0.2
* Update iPhoneOS SDK 5.1 -> 8.1
* Transaction tracking not implemented/disabled (time constraints)

Note: to avoid collision with Flurry ANE, some functions were renamed.

This shouldn't be necessary but unfortunately "-hideAneSymbols yes"
seems
broken in AIR right now.

* Renamed logEvent -> FRE_logEvent
* Renamed setAppVersion -> GA_setAppVersion (avoid collision with Flurry
* ANE)
* Renamed getVersion -> GA_getVersion

Note: These files are required before building:

cp '/Applications/Apache
Flex/SDK/SDK-4.9.1/include/FlashRuntimeExtensions.h'
source/ios/Libraries

cp '/Applications/Apache
Flex/SDK/SDK-4.9.1/lib/android/FlashRuntimeExtensions.jar'
source/android/libs

Commit:
Add binary

Commit:
Recompile with java 1.6 (ArgumentError on Android)

Commit:
Fix function names in GAContext for Android

cedtat commented:
Great work! This is what I am searching for. 
Could it be possible to provide an ANE package of your release ?

... oh, I found your bin directory in another branch!

Commit:
bug: get version() calls the wrong extension function.

Commit:
Updated startNewSession, setCustomDimension, clearCustomDimension, setCustomMetric, clearCustomMetric to GA 3.0.

Note: "startNewSession" starts a new session with sending a UX: appstart event.

Bug: "trackView" did not set screen name. Merged "setScreenName" into "trackView".

mparrett committed:
Merge pull request #1 from nostek/update-to-v3-and-64bit-upstream

Update to v3 and 64bit upstream
